dms/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "13.0.6.0.0",
+    "version": "13.0.7.0.0",
```

Comment:
Don't change the major version, but minor one. The change is not of such entity.